sandbox/focus_keybindings.py

```diff
@@ -0,0 +1,54 @@
+from textual.app import App
+from textual.widget import Widget
+from textual.widgets import Static
+
+
+class FocusKeybindsApp(App):
+    dark = True
+
+    def on_load(self) -> None:
+        self.bind("1", "focus('widget1')")
```

Comment:
That's indeed a typical use case of PR #543 ! :slightly_smiling_face: 
Once it's merged this code could be rewritten into something like this:
```py
    def on_load(self) -> None:
        target_keys = ("1", "2", "3", "4", "q", "w", "e", "r")
        self.bind(",".join(target_keys), "widget_key('$event.key')")

    def action_widget_key(self, key: str) -> None:
        self.focus(f"widget{key}")

Reply:
it seems that #543 won't be merged before a while, though, so I guess we'll have to merge this one without it as a first iteration :slightly_smiling_face: